internal/version/version.go

```diff
@@ -0,0 +1,23 @@
+package version
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"time"
+)
+
+var Version string = "0.0.0"
+var GitCommit string = "unknown"
+var BuildTime string = "unknown"
+
+func ToString() string {
```

Comment:
A total nit but the name ToString sounds like a Javascript convention. The go-idiomatic way of naming it would be just String, in analogy with fmt.Stringer (aka the String method), flag.String, and so on.